Realm/Realm/Realm.cs

```diff
@@ -785,6 +785,38 @@ public T Write<T>(Func<T> function)
             return result;
         }
 
+        /// <summary>
+        /// Factory for a write <see cref="Transaction"/>. Essential object to create scope for updates.
+        /// </summary>
```

Comment:
The original docs are kind of vague, so rather than copying them, I suggest we rework them. You may want to update the `BeginWrite` docs to be more helpful as well.

```suggestion
        /// <summary>
        /// Asynchronously begin a write transaction for this Realm.
        /// </summary>
        /// <remarks>
        /// This method asynchronously acquires the write lock and then dispatches the continuation on the original
        /// thread the Realm was opened on. The transaction can then be committed either asynchronously or
        /// synchronously.
        /// <br/>
        /// When invoked on a thread without SynchronizationContext (i.e. typically background threads), this method
        /// calls <see cref="BeginWrite"/> and executes synchronously.
        /// </remarks>
```